### ISSUE TITLE
Add a page on using GOV.UK Pay in your prototype

### ISSUE THREAD

**Issue by m-green** [alphagov/govuk-prototype-kit-docs#137]
In the Prototype Kit documentation, there's guidance on how to use [Verify](https://govuk-prototype-kit.herokuapp.com/docs/using-verify) and [Notify](https://govuk-prototype-kit.herokuapp.com/docs/using-notify) in your prototype - but not Pay.

There's some information in the [Testing your integration page](https://docs.payments.service.gov.uk/testing_govuk_pay/#testing-your-integration) of the Pay tech docs that might be useful. But would definitely also need a conversation with a Pay dev to understand whether/how to use Pay test/live accounts in a prototype.

(Thanks to @joelanman for raising this.)

**Comment by NickColley** [comment]
We should consider asking GOV.UK Pay if they want to contribute this in the future, or work in collaboration with them.

**Comment by joelanman** [comment]
I dont think this is a plugin, I think Pay have demo pages you can connect your prototype to. We should talk to them
